### DISCUSSION TITLE
Should pagination work for a blog?

### DISCUSSION THREAD

**Discussion by jimmbelll** [imfing/hextra#372] [Q&A]
I imported 90+ blog entries into my Hextra site under /blog, and that landing page shows them all as snippets, but all 90+ on that page. The links to single pages work fine, as well as the previous/next links at the bottom of the single page.

I see pagination in Hugo, but I can't tell if Hextra has it. I'm studying layouts/blog/list.html and it seems not to, but does it somehow inherit it from Hugo?  If not, does someone have a recipe or example that paginates?

Thanks for bringing us this really great theme!

**Comment by imfing** [comment]
Hi @jimmbelll 

Unfortunately, pagination hasn't been implemented yet for blog listing page. 

I'll add this as an issue and work on it to include it in a future update.

> does it somehow inherit it from Hugo? If not, does someone have a recipe or example that paginates?

You might want to search online to see how other themes implement it. It’s usually not too hard to override Hextra’s blog listing template to include this.
